package.json

```diff
@@ -428,15 +441,15 @@
     "embedding-sdk:publish": "cd ./resources/embedding-sdk && npm publish",
     "embedding-sdk:generate-nextjs-compat": "tsx enterprise/frontend/src/embedding-sdk/bin/generate-nextjs-compat.ts",
     "embedding-sdk:test-unit": "yarn test-unit enterprise/frontend/src/embedding-sdk/",
-    "embedding-sdk:tsc": "tsc --project tsconfig.sdk.json",
+    "embedding-sdk:tsc": "tsc --project tsconfig.sdk.json || true",
```

Comment:
What is the `|| true` for? I think this makes the exit code always `0`

Reply:
This needed to not fail on type errors during the .d.ts generation.
For webpack we had a special hack to convert errors to warnings, here we just don't fail with forcing exit code 0

The `yarn embedding-sdk:tsc` is used only for dts generation from what i see, let's rename `yarn embedding-sdk:tsc` to `yarn embedding-sdk:dts` to be more explicit about how it's used